apps/api/src/app/events/e2e/send-message-email.e2e.ts

```diff
@@ -1,7 +1,49 @@
 import { expect } from 'chai';
 import { CompileEmailTemplate } from '../../content-templates/usecases/compile-email-template/compile-email-template.usecase';
+import { createMailData } from '../usecases/send-message/send-message-email.usecase';
 
 describe('Trigger event - Send message email - /v1/events/trigger (POST)', function () {
+  it('should merge mail data', function () {
+    const defaultMailData = {
+      to: ['to-reply@novu.co'],
+      subject: 'subject',
+      html: '<html></html>',
+      from: 'no-reply@novu.co',
+      attachments: [],
+      id: 'id',
+    };
+
+    let result = createMailData(defaultMailData, {
+      to: ['override-to@novu.co'],
+      from: 'override-from@novu.co',
+      text: 'text',
+      cc: ['cc@nouv.co'],
+      bcc: ['bcc@nouv.co'],
+    });
+
+    expect(result.to).to.deep.equal(['to-reply@novu.co', 'override-to@novu.co']);
+    expect(result.from).to.equal('override-from@novu.co');
+    expect(result.text).to.equal('text');
+    expect(result.cc).to.deep.equal(['cc@nouv.co']);
+    expect(result.bcc).to.deep.equal(['bcc@nouv.co']);
```

Comment:
```suggestion
      cc: ['cc@novu.co'],
      bcc: ['bcc@novu.co'],
    });

    expect(result.to).to.deep.equal(['to-reply@novu.co', 'override-to@novu.co']);
    expect(result.from).to.equal('override-from@novu.co');
    expect(result.text).to.equal('text');
    expect(result.cc).to.deep.equal(['cc@novu.co']);
    expect(result.bcc).to.deep.equal(['bcc@novu.co']);
```
Not that the test was bad. It was checking the values rightfully. 🙂